src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -329,7 +329,7 @@ type UseBalanceChartDataReturn = {
 }
 
 type UseBalanceChartDataArgs = {
-  assetIds: AssetId[]
+  assetId?: AssetId
   accountId?: AccountId
```

Comment:
the balance chart hook now behaves like all other selectors - takes optional asset and account ids as filters. 
* without either, it's the dashboard
* with just an asset id, it's the aggregated balance across all accounts for that asset
* with just an account id, it's all assets within that account
* with both an asset and account id, it's that specific combo